js/src/gallery-block/index.js

```diff
@@ -39,22 +39,46 @@ registerBlockType( 'cloudinary/gallery', {
 	category: 'widgets',
 	icon: 'format-gallery',
 	attributes: {
-		displayProps_mode: { type: 'string', default: 'classic' },
-		displayProps_columns: { type: 'number', default: 1 },
-		themeProps_primary: { type: 'string', default: primaryColor },
+		displayProps_mode: {
+			type: 'string',
+			default: 'classic',
+		},
+		displayProps_columns: {
+			type: 'number',
+			default: 1,
+		},
+		themeProps_primary: {
+			type: 'string',
+			default: primaryColor,
+		},
 		themeProps_onPrimary: {
```

Comment:
@dugajean 

Let's remove this property as it doesn't seem to be used anywhere.